app-script-payslip-sender/PayslipSender.js

```diff
@@ -0,0 +1,26 @@
+function PayslipSender() {
+  var spreadsheet = SpreadsheetApp.getActive().getSheetByName('List');
+  var dataRange = spreadsheet.getRange("A2:D6").getValues();
```

Comment:
what if the sheet has more than just six rows? Need to handle that case as well.